ERCS/erc-7652.md

```diff
@@ -0,0 +1,160 @@
+    );
+
+    /// @notice           Emitted when `Guarantee sequence` is established for an NFT
+    /// @param userGuaranteed      address of guaranteed
+    /// @param number  block.number of transaction,
+    ///                and all DAOs established before this point will enter the guarantee sequence
+    /// @param DAOs   DAO sequence providing guarantee
+    /// @param tokenId Guaranteed NFT (token ID),
+    event GuaranteeSequenceIsEstablshed(
+        address userGuaranteed,
+        uint256 number,
+        address DAOs,
+        uint256 indexed tokenId
+    );
+
+    /// @notice   A user's evaluation for an NFT (token ID)
+    /// @dev      Set the guarantee information for one guarantor,
+    /// Throws if `_tokenId` is not a valid NFT
+    /// @param value  user's evaluation for  an NFT, the oledr value is canceled,
+    /// @param user   address of guarantor
+    /// @param weight guarantee weight for guarantor
+    /// @param tokenId The NFT
+    /// @return the error status of function execution
+    function setNFTGuarantedInfo(
+        uint256 value,
+        address user,
+        uint256 weight,
+        uint256 tokenId
+    ) external returns (uint256);
+
+    /// @notice   Establish guarantee sequence for an NFT (token ID) and split the commission
+    /// @dev      Each NFT(token ID) retains a current guarantee sequence,
+    ///           and expired guarantee sequences are no longer valid,
+    ///           Throws if `_tokenId` is not a valid NFT
+    /// @param valueCommission Commission for a transactions
+    /// @param userGuaranteed   address of guaranteed
+    /// @param number  block.number of transaction,
+    ///              and all DAOs established before this point will enter the guarantee sequence
+    /// @param tokenId The NFT
+    /// @return the error status of function execution
+    function establishNFTGuarantee(
+        uint256 valueCommission,
+        address userGuaranteed,
+        uint256 number,
+        uint256 tokenId
+    ) external returns (uint256);
+
+    /// @notice   Transactions that fulfill the guarantee responsibility
+    /// @dev      The new accountability transaction also requires
+    ///           the construction of a new guarantee sequence
+    ///           Throws if `_tokenId` is not a valid NFT or userGuaranteed is not right
+
+    /// @param  userGuaranteed   address of guaranteed
+    /// @param  tokenId The NFT
+    /// @return the error status of function execution
+    function FulfillGuaranteeTransfer(address userGuaranteed, uint256 tokenId)
+        external
+        returns (uint256);
+}
+
+```
+
+## Rationale
+
+Key factors influencing the standard:
+
+- Pay attention to ensuring fairness between and within groups when allocating commissions
+- Keeping the number of guarantee groups (DAOs)in the interfaces to prevent contract bloat
+- The guarantee group is a DAO contract, which MUST implement the `ERC721TokenReceiver` interface
+- Simplicity
+- Gas Efficiency
```

Comment:
You should go in-depth on specific choices you made, not give general thoughts on your design.

Reply:
Thank you for your suggestion,
These descriptions are issues that should be focuses on in the coding implementation.
The reference implementation will also pay attention to these issues.